circuit-lib/circuit-lib.circom/package.json

```diff
@@ -6,30 +6,34 @@
   "scripts": {
     "build-masp": "sh scripts/buildCircuit.sh Masp",
     "build-app": "sh scripts/buildCircuit.sh App",
-    "test-light-circuits": "ts-mocha --resolveJsonModule ./tsconfig.json -t 100000000 tests/**.ts --exit",
+    "test-light-circuits": "ts-mocha --resolveJsonModule ./tsconfig.json -t 100000000 tests/{*.test.ts,**/*.test.ts} --exit",
     "test-build-light-circuits": "yarn build-all && ./scripts/checkBuildCircuit.sh",
+    "build": "yarn tsc",
     "test": "yarn test-light-circuits && yarn test-build-light-circuits",
-    "format": "prettier --write \"tests/**/*.{ts,js,circom}\"",
     "build-all": "yarn build-app 4 && yarn build-masp 2 && yarn build-masp 10",
-    "lint": "yarn prettier \"tests/**/*.{ts,js}\" --check",
-    "build": "yarn tsc"
+    "lint": "yarn prettier \"tests/**/*.{ts,js}\" \"src/**/*.{ts,js}\" --check",
+    "format": "prettier --write \"tests/**/*.{ts,js}\""
   },
   "author": "",
   "devDependencies": {
+    "@lightprotocol/cli": "0.1.1-alpha.21",
+    "@lightprotocol/zk.js": "0.3.2-alpha.15",
     "@types/chai": "^4.3.0",
+    "@types/chai-as-promised": "^7.1.6",
     "@types/mocha": "^10.0.1",
     "@types/node-fetch": "^2.6.2",
     "chai": "^4.3.7",
-    "@lightprotocol/zk.js": "0.3.2-alpha.15",
+    "chai-as-promised": "^7.1.1",
+    "circom_tester": "^0.0.19",
+    "circomlib": "^2.0.5",
     "mocha": "^10.2.0",
+    "prettier": "^2.6.2",
     "ts-mocha": "^10.0.0",
     "typescript": "^4.3.5",
-    "prettier": "^2.6.2",
-    "typescript-collections": "^1.3.3",
-    "circomlib": "^2.0.5"
+    "typescript-collections": "^1.3.3"
   },
   "main": "lib/index.js",
   "files": [
     "src/**/*"
   ]
```

Comment:
why are there so many changes in this file which aren't actual changes?
(Makes it difficult to spot the actual changes, thus difficult to review.)

Reply:
yea, package.json was reorganized, I can limit some lines of changes by reorganizing again. But overall the PR has gone pretty large especially after readapting to be test-functional